i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-get-report.mdx

```diff
@@ -0,0 +1,115 @@
+# How to Get a Test Run Report
+
+## testplane Report {#testplane_report}
+
+After completing all test runs, testplane writes the result to the console in the form of a string like this:
```

Comment:
Our team member would say "Testplane as product name should be written with upper case T, and it could be written with lower case t if it is used as cli command".

I suggest to quickly fix this everywhere replacing " testplane " with " Testplane ". This should fix most of these.